client/jetpack-cloud/sections/sidebar-navigation/manage-selected-site.tsx

```diff
@@ -131,6 +132,15 @@ const useMenuItems = ( {
 					enabled: isAdmin && isSectionNameEnabled( 'jetpack-social' ) && ! isWPForTeamsSite,
 					isSelected: itemLinkMatches( path, `${ JETPACK_CLOUD_SOCIAL_LINK }/${ siteSlug }` ),
 				},
+				{
+					icon: <JetpackIcons icon="subscribers" size={ 24 } />,
```

Comment:
Did you try using WP icon directly instead?

```jsx
import { Icon, people } from '@wordpress/icons';

<Icon icon={ people } size={ 24 } />
```

Reply:
I did not. Would that be better? I'll update it if that's preferred.